lisa/testsuite.py

```diff
@@ -65,8 +66,10 @@ def check_environment(
         check_result = requirement.environment.check(environment.capability)
         if check_result.result and requirement.os_type and environment.is_ready:
             for node in environment.nodes.list():
+                # use __mro__ to match any super types.
+                # for example, Unbuntu satisifies Linux
```

Comment:
```suggestion
                # for example, Ubuntu satisifies Linux
```

Nice trick of using the class hierarchy as a fallback mechanism!